analytics-test.rb

```diff
@@ -0,0 +1,29 @@
+require 'FileUtils'
+
+puts "Enter the filetype you want to search (rb, erb, md, or html)"
+filetype = gets.chomp.to_s
+
+puts "Processing #{filetype} files recursively in #{Dir.pwd} with Rorybot"
+`rorybot **/*.#{filetype} | tee rorybot_output.txt`
+
+input_file = File.open('rorybot_output.txt', 'r')
+output_file = File.open('frequency.csv', 'w+')
+warning_message = /(?:warning\s{2,}?)(.+?)(?=\s{2,})/
```

Comment:
`^.*warning  (.*)$`

- `^` denotes the beginning of the line.
- `.*` matches everything _up to_ `warning`
- `warning` matches `warning`
- `  ` matches the 2 spaces (could also be `\s+`)
- `(.*)$` creates a capture group for the rest of the line because `$` means "end of line"